tools/export-strings.sh

```diff
@@ -15,15 +15,24 @@ fi
 echo "[*] Cloning mozilla-l10n/focusios-l10n"
 git clone https://github.com/mozilla-l10n/focusios-l10n.git
```

Comment:
You might want to add a check to remove the folder like you did for tools/Localizations, or at least skip cloning if the folder exists. Right now, it would just fail if the folder already exists.

Same for the import script.